webapp/channels/src/components/gif_picker/components/gif_picker_items.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {memo, useMemo} from 'react';
+import {GiphyFetch} from '@giphy/js-fetch-api';
+import {EmojiVariationsListProps, Grid} from '@giphy/react-components';
+
+const giphyFetch = new GiphyFetch('DxfEk2t4CR8bv4A1kviDBRMP9i1og3Da');
```

Comment:
Is this the prod key? Is this ok to use for test environment as well or should we support a test api key  as well?

Reply:
this is the development key, i will replace it with taken from env

Reply:
Ah perfect, sounds good 👍